### PR TITLE
DSPHLE: fix problems with older ucodes

### PR DESCRIPTION
**Pull request by Tilka**
This fixes https://bugs.dolphin-emu.org/issues/12515.

### CODE REVIEW

**Comment by Rumi-Larry** [comment]
Above on the lines 246-248 where it says:
`  // Capcom vs. SNK 2, Naruto 2, Lost Kingdoms, Star Fox, Mario Party 4, Mortal Kombat,
  // Smugglers Run Warzone, Smash Brothers, Sonic Mega Collection, ZooCube
  // nddemo, Star Fox`
There seems to be some issues with the indentation. If some uCodes are supposed to be there, then some kind of placeholder should be placed.

**Comment by Rumi-Larry** [comment]
Also, on line 234 there's this `//CARD` comment that's missing from the other cases like GBA or ROMS. I think it can be removed.

File: Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp [hunk 1]
```diff
@@ -214,7 +214,7 @@ void AXWiiUCode::HandleCommandList()
 
       case CMD_OUTPUT:
       case CMD_OUTPUT_DPL2:
-        volume = m_cmdlist[curr_idx++];
+        volume = m_crc == 0xd9c4bf34 ? 0x8000 : m_cmdlist[curr_idx++];
```
Comment (leoetlino) [inline Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp:217]: Where does this value come from? Is it hardcoded in that particular µcode?

**Comment by leoetlino** [comment]
@Rumi-Larry while the formatting should probably be fixed (in a separate PR), those changes are not relevant to this PR at all.

File: Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp (hunk 1 above)
Comment (Tilka) [reply Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp:217]: 0x8000 is full volume. I'm just copying the behavior of CMD_OUTPUT_OLD. I haven't independently verified that this is correct.

Comment (leoetlino) [reply Source/Core/Core/HW/DSPHLE/UCodes/AXWii.cpp:217]: CMD_OUTPUT in µcode 0xd9c4bf34:

image

Seems to match what we do in OutputSamples (the whole clamping loop is a no-op considering the volume never changes from 0x8000).

**Review by leoetlino — APPROVED** [review]

**Pull request merged**